doc/usage.rst

```diff
@@ -98,6 +98,12 @@ few command line options:
    to parse ``setup.py``. This flag can be provided multiple times to
    pass multiple packages for pre-install.
 
+.. cmdoption:: --pip-tool <exename>
+
+   Executable that will be used to install requirements after the
+   preinstall stage.  Usually you'll install this program by using the
+   ``--preinstall`` argument. By default vanilla pip is used.
```

Comment:
Sounds like good ideas to me, I'll update the text here to be more clear about the location of the binary and remove the bit about vanilla pip